alfasim_sdk/status.py

```diff
@@ -18,8 +23,13 @@ class ErrorMessage:
 @attr.s(kw_only=True)
 class WarningMessage:
     """
-    The model_name and the message informed is displayed on the status monitor of ALFAsim.
-    The execution of a new simulation will be kept normal.
+    WarningMessage allows the plugin to display a message to the user over the status monitor, and signalize a minor
+    issue, that needs to be fixed but doesn't need to lock the simulation.
```

Comment:
Remove extra comma and two fixes:

WarningMessage allows the plugin to display a message to the user over the status monitor, and _signalizes_ a minor issue that needs to be fixed but doesn't need to _block_ the simulation.